website/content/docs/job-specification/check.mdx

```diff
@@ -369,26 +369,32 @@ For checks registered into the Nomad service provider, the status information of
 checks can be viewed per-allocation. The `alloc status` command now includes
 summary information for Nomad service checks.
 
-```
-➜ nomad alloc status <allocation-id>
+```shell-session
+$ nomad alloc status <allocation-id>
 ```
 
-```
+<CodeBlockConfig hideClipboard>
```

Comment:
does this just get rid of the "Copy" button in the upper-right corner of the code block?

Reply:
Yep.  Doesn't seem to make sense to copy example command _output_